src/ui/src/js/controllers/command_index.js

```diff
@@ -41,9 +52,10 @@ export default function commandIndexController(
     let breadCrumbs = [];
 
     response.data.forEach((system) => {
-      system.commands.forEach((command) => {
+      system.commands.forEach((command) => 
```

Comment:
You forgot to add the open bracket here. I am getting an error when I try to compile the UI. When I put the bracket back in it compiles.

```
SyntaxError: /home/gnburch/PycharmProjects/beer-garden-v3/src/ui/src/js/controllers/command_index.js: Unexpected token, expected "," (64:10)

  62 |           version: system.version,
  63 |           description: command.description || 'No Description Provided',
> 64 |         });
     |           ^
  65 |       });
  66 |     });
  67 | 

```